src/step/issues/gitea.rs

```diff
@@ -0,0 +1,60 @@
+use super::Issue;
+pub(crate) use crate::integrations::gitea::ListIssuesError as Error;
+use crate::{
+    integrations::gitea::{list_issues, ListIssuesError},
+    prompt,
+    state::{self, RunType, State},
+};
+
+pub(crate) fn select_issue(
+    labels: Option<&[String]>,
+    run_type: RunType,
+) -> Result<RunType, ListIssuesError> {
+    match run_type {
+        RunType::DryRun { state, .. } if state.gitea_config.is_none() => {
+            Err(ListIssuesError::NotConfigured)
+        }
+        RunType::DryRun {
+            mut state,
+            mut stdout,
+        } => {
+            if let Some(labels) = labels {
+                writeln!(
+                    stdout,
+                    "Would query configured Gitea instance for issues with labels {}",
+                    labels.join(", ")
+                )?;
+            } else {
+                writeln!(
+                    stdout,
+                    "Would query configured Gitea instance for issues with any labels"
+                )?;
+            }
+
+            writeln!(
+                stdout,
+                "Would prompt user to select an issue and move workflow to IssueSelected state."
```

Comment:
```suggestion
                "Would prompt user to select an issue."
```

That state transition concept is a holdover from an older version—I probably need to clean that up in a few places still